keyboards/dztech/dz60/readme.md

```diff
@@ -10,8 +10,14 @@
 * Hardware Supported: DZ60 60% Mechanical Keyboard PCB
 * Hardware Availability: [KBDfans](https://kbdfans.com/collections/60/products/dz60-60-pcb)
 
+There are two versions of the DZ60 PCB. Please use the appropriate firmware for your board.
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```